src/core_ocean/Registry.xml

```diff
@@ -391,8 +391,27 @@
 					description="Vertical component of Redi mixing limited in bottom config_Redi_surface_layer_tapering_depth"
 					possible_values="Any positive real value, typically between 100 and 500 m."
 		/>
-		<nml_option name="config_standardGM_tracer_kappa" type="real" default_value="0.0" units="m^2 s^{-1}"
-					description="Coefficient of standard GM parametrization of eddy transport (Bolus component), $\kappa$"
+		<nml_option name="config_GM_Bolus_kappa_function" type="character" default_value="constant" units="unitless"
+					description="Control what type of function is used for GM Bolus kappa"
+					possible_values="'constant' and 'ramp'"
+		/>
+		<nml_option name="config_standardGM_tracer_kappa" type="real" default_value="600.0" units="m^2 s^{-1}"
+					description="Coefficient of standard GM parametrization of eddy transport (Bolus component), $\kappa$. Used when config_GM_Bolus_kappa_function is set to constant."
+		/>
+		<nml_option name="config_GM_Bolus_kappa_min" type="real" default_value="0.0" units="m^2 s^{-1}"
+					description="Coefficient of standard GM parametrization of eddy transport (Bolus component), $\kappa$, min value in ramp as a function of dcEdge. Used when config_GM_Bolus_kappa_function is set to ramp."
+					possible_values="Any positive real value."
+		/>
+		<nml_option name="config_GM_Bolus_kappa_max" type="real" default_value="600.0" units="m^2 s^{-1}"
+					description="Coefficient of standard GM parametrization of eddy transport (Bolus component), $\kappa$, min value in ramp as a function of dcEdge. Used when config_GM_Bolus_kappa_function is set to ramp."
+					possible_values="Any positive real value."
+		/>
+		<nml_option name="config_GM_Bolus_cell_size_min" type="real" default_value="20e3" units="m"
+					description="Minimum value in grid cell size for GM $kappa$ ramp function.  Here cell size refers to dcEdge. Used when config_GM_Bolus_kappa_function is set to ramp."
+					possible_values="Any positive real value."
+		/>
+		<nml_option name="config_GM_Bolus_cell_size_max" type="real" default_value="30e3" units="m"
+					description="Maximum value in grid cell size for GM $kappa$ ramp function.  Here cell size refers to dcEdge"
```

Comment:
Do we think that 20 to 30 km is the range over which we want GM to turn on in practice?  That seems surprisingly narrow to me but I could be wrong.  The deformation radius around Antarctica is tiny -- order 5 to 9 km -- so maybe I have a skewed perspective.

Reply:
Good point. I didn't choose it too carefully.  The span of the ramp is something we need to test.

I thought that, if 1/3 degree needs GM on, but 1/10 has GM off, then switch should be somewhere between 10 and 30 km. See more discussion below.

Reply:
Short of doing a local Rossby Radius dependency, I think that this is the best compromise (20-30 km range). For example, the metoffice is not using GM at all for their latest 0.3deg simulations, as far as I understand. I agree that this is not ideal for the high-latitudes, but seems good for these simplified settings.
@maltrud: what do you think about this?

Reply:
This is definitely something that we'll need to test. The idea is to help testing along using the E3SM-ARRM (Arctic refined) configuration, where the lower resolution grid is EC60to30 in the Southern hemisphere and 30to10 in the Northern hemisphere. This grid has already been run for ~10 years or so without GM.